src/visualizer/visualizer.js

```diff
@@ -108,7 +122,7 @@ const Visualizer = () => {
     }, []);
 
     if (loading) {
-        return <div />
+        return <Welcome />
```

Comment:
Please keep this as `<div />` . We don't wont it to be like jump in around from `Welcome` to `Visualizer` since there's a slight delay

Reply:
Done.